### DISCUSSION TITLE
UI: Home screen minor styling fix

### DISCUSSION THREAD

**Discussion by swedishfrenchpress** [cashubtc/eNuts#235] [General]
Just a small suggestion for some styling tweaks on the home screen. Loving the feature where you can easily hide by tapping the eNuts logo, pretty great.

- I've noticed an inconsistency issue related to wording. The term changes from "see full transaction history" to "transaction history," accompanied by a list icon on the left. I suggest aligning these terms for consistency.
- Additionally, I suggest the removal of the white line. While this suggestion is purely aesthetic, I believe it aligns with Dieter Rams' principle: "Good design is as little design as possible." Just my two sats - you are free to agree or disagree with this suggestion :).

![current](https://github.com/cashubtc/eNuts/assets/78821053/51e63928-123c-4eef-952b-c0ad0cefe245)
![prop](https://github.com/cashubtc/eNuts/assets/78821053/99a860b5-db0e-4ede-94b3-6df8a8ed2345)

**Comment by KKA11010** [comment]
Great suggestions as always

**Comment by KKA11010** [comment]
https://github.com/cashubtc/eNuts/issues/237
